recipes/syntok/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "syntok" %}
+{% set version = "1.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4e10ed25e5cd131138fe75ae39917f6d70deab07da57726564f602daa2d5303e
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python =2.7|>=3.5
```

Comment:
Do you actually need the Python 2.7 support? Upstream has removed it from the package metadata and 2.7 is not supported overall by conda-forge.

Reply:
No not really, I just wanted to include all possible versions. But you're right, some parts of the docs have removed it so I'll update it accordingly.